docs/quickstart/dind.md

```diff
@@ -534,3 +535,25 @@ There are a couple of caveats to keep in mind here:
 
     -   This behavior differs from when Docker volume mounts are mounted into
         the system container's `/var/lib/docker` (see prior section).
+
+## Caching Docker Images among multiple Docker-in-Docker Instances
+
+Running Docker inside system containers leads to the situation where multiple
+Docker instances are running on the same host machine (i.e., each instance
+running inside a system container).
+
+A common need in this scenario is to have a local Docker image cache that can be
+shared among those Docker instances, to reduce network traffic generated by the
+Docker daemons pulling those images from a remote repository (e.g., Dockerhub).
+
+To achieve this, one may be tempted to create a single host directory or volume
+that is mounted into the `/var/lib/docker` of all the system containers that are
+running Docker. But this is not correct because each Docker instance must each
+have a dedicated `/var/lib/docker` data-store (i.e., it can't be shared).
```

Comment:
Add a reference here to the other doc where we mention the docker-limitation that prevents this.